dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -80,10 +80,8 @@ def test_numerical_stats_option(self, *mocks):
                 self.assertIsNone(profile_column["statistics"]["max"])
                 self.assertEqual(0, profile_column["statistics"]["variance"])
                 self.assertIsNone(profile_column["statistics"]["quantiles"][0])
-                self.assertTrue(
-                    profile_column["statistics"]["skewness"] is np.nan)
-                self.assertTrue(
-                    profile_column["statistics"]["kurtosis"] is np.nan)
+                self.assertEqual(profile_column["statistics"]["skewness"], 0)
+                self.assertEqual(profile_column["statistics"]["kurtosis"], 0)
```

Comment:
the standard we try to commit to is we do the `assert(expected_value, actual_value)` hence we just need to flip these around.

We should have a convo with @AnhTruong @lettergram about the implication of this being 0 and not `nan`